Documentation/HowTo/adding-sections-and-pages.md

```diff
@@ -0,0 +1,61 @@
+# Adding Sections and Pages
+
+Publish works with two main types of pages: Sections and Pages.
+
+A **Section** is a top-level page which shows up in the menu.
+A **Page** is a stand-alone page which you can assign any URL. These do not show up in the menu.
+
+## Creating a new Section
+
+To create your sections, open up the main.swift file and fill in each section inside the `SectionID` enum like below:
+```
+struct MyWebsite: Website {
+  enum SectionID: String, WebsiteSectionID {
+    // Add the sections that you want your website to contain here:
+    case blog
+    case about
+  }
+}
+```
+
+### Adding content to your section-page
+
+To add content to these sections, you have a few options: markdown or manual Swift-code. To create your section using markdown, you must create a folder inside the Content-folder with the same name as your enum case. In the case above, you should create the folders: `/Content/articles` and `/Content/about-me`.
```

Comment:
I think the folders should be as in your example:

> In the case above, you should create the folders: `/Content/blog` and `/Content/about`.

Reply:
Or maybe not.. Haha
It of course depends on how you want the url for your site to be. But I think the last line then don't make sense. 

> you must create a folder inside the Content-folder with the same name as your enum case. In the case above, you should create the folders: `/Content/articles` and `/Content/about-me`.
This part is confusing

Reply:
> I think the folders should be as in your example:
> 
> > In the case above, you should create the folders: `/Content/blog` and `/Content/about`.

I think you're right. Will fix.

Tak ;)